.github/workflows/sphinx.yml

```
@@ -63,52 +63,4 @@ jobs:
-      - name: Make HTML frontend
-        run: poetry run sphinx-build docs/source/ docs/build/ -a
-      - name: Deploy
-        uses: peaceiris/actions-gh-pages@v4
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_branch: gh-pages  
-          publish_dir: ./frontend/docs/build
-          destination_dir: docs/frontend
-          keep_files: true
\ No newline at end of file
+ 
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)